byzcoin/statetrie_test.go

```diff
@@ -71,3 +82,125 @@ func TestStateTrie(t *testing.T) {
 	require.NoError(t, sst.Commit())
 	require.True(t, bytes.Equal(sst.GetRoot(), newRoot))
 }
+
+// TestDarcRetrieval is used as a benchmark here. It stores a lot of darcs
```

Comment:
Then, use `testing.B` ;)

Reply:
But I'm smarter than go ;) I wanted to be able to change all three of `depth`, `width` and `entries`, which is not possible with a benchmark.